script.js

```diff
@@ -1,4 +1,103 @@
+var myOutputValue = "";
+var userWin = 0;
+var computerWin = 0;
+var totalPlay = 0;
+var currentGameMode = "Waiting for username";
+var userName = "";
+
+// Computer generates random Number
+var randomNumber = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  return randomInteger;
+};
+
+//Computer assigns number to "scissors", "paper", or "stone"
+var computerGuesses = function () {
+  var computerGuess = "";
+  var guess = randomNumber();
+  if (guess == 0) {
+    computerGuess = "scissors";
+  }
+  if (guess == 1) {
+    computerGuess = "paper";
+  }
+  if (guess == 2) {
+    computerGuess = "stone";
+  }
+  return computerGuess;
+};
```

Comment:
You can consider using else if statements over here. 
If with else if statements will stop checking for a true condition once they find one but if with if statement will still run through all the if statements even after it has found a true statement. 

Since there is only one number generated, once the relevant if statement is found to be true, we don't need the rest of the statements to be checked or run. So in this case else if statements would be more accurate. 

https://www.w3schools.com/js/js_if_else.asp